src/system/BlocksModule/Container/LinkContainer.php

```diff
@@ -87,8 +89,7 @@ private function getAdmin()
                 'text' => $this->translator->__('Create new block'),
                 'icon' => 'plus'
             ];
-        }
-        if ($this->permissionApi->hasPermission('ZikulaBlocksModule::', '::', ACCESS_ADD)) {
```

Comment:
This line is removed because it was checked twice.
```
if ($this->permissionApi->hasPermission('ZikulaBlocksModule::', '::', ACCESS_ADD)) {
// A
}
if ($this->permissionApi->hasPermission('ZikulaBlocksModule::', '::', ACCESS_ADD)) {
// B
}
```
became
```
if ($this->permissionApi->hasPermission('ZikulaBlocksModule::', '::', ACCESS_ADD)) {
// A
// B
}
```